lib/stores/S3Store.ts

```diff
@@ -123,9 +128,12 @@ export default class S3Store extends DataStore {
       request.Metadata.original_name = parsedMetadata.filename.encoded
     }
 
+    // TODO: rename `file` to `upload` to align with the codebase
```

Comment:
Would addressing this TODO break existing uploads? i.e. a new version of tus-node-server would not be able to read upload's that were created by older tus-node-server versions?

Reply:
That's correct and already the case actually on the `1.x` branch. But AFAIK that only matters for GET requests, which is not part of the core protocol. Should we do something about that?

Reply:
If a fix is as simple as checking for the `file` or `upload` key in the metadata, I think we could implement it.

Reply:
So I was mistaken and by  "That's correct and already the case actually on the `1.x` branch" I meant that the keys were already renamed inside the object. So it's not as simple as that. Perhaps I should rename them back (such as `size` to `upload_length`).